Automatic Validation ended with: 
> 2026-01-21T17:19:20.4786068Z ##[error] Manifest Error: Schema Error. [Schema validation failed. Error context: <root>[InstallationMetadata] Description: Value type not permitted by 'type' constraint. Error context: <root> Description: Failed to validate against schema associated with property name 'InstallationMetadata'. ] File: dongdongbh.Mindwtr.installer.yaml Manifest Error: Encountered unexpected root node.

(Automated response - build 1200.)

--- a/manifests/d/dongdongbh/Mindwtr/0.5.7/dongdongbh.Mindwtr.installer.yaml
+++ b/manifests/d/dongdongbh/Mindwtr/0.5.7/dongdongbh.Mindwtr.installer.yaml
@@ -17,7 +17,7 @@ ReleaseDate: 2026-01-21
 AppsAndFeaturesEntries:
   - ProductCode: Mindwtr
 InstallationMetadata:
-  - DefaultInstallLocation: '%LOCALAPPDATA%\Programs\Mindwtr'
+  DefaultInstallLocation: '%LOCALAPPDATA%\Programs\Mindwtr'
 Installers:
   - Architecture: x64
     InstallerUrl: https://github.com/dongdongbh/Mindwtr/releases/download/v0.5.7/Mindwtr_0.5.7_x64-setup.exe

